<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1281</width>
      <height>721</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../img/bg1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>150</y>
      <width>551</width>
      <height>421</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0,0,0,200);
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>150</y>
      <width>361</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 99, 177);
font: 700 22pt &quot;.VnAvant&quot;;</string>
    </property>
    <property name="text">
     <string>Create an account</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_name">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>antipasto</family>
      <pointsize>15</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;antipasto&quot;;
background-color:rgba(0, 0, 0, 0); border:none;
border-bottom: 2px solid rgba(0, 99, 177, 255); color:rgba(156, 194, 213, 230);
padding-bottom:7px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_register">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>460</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_register{
	color: rgb(0, 99, 177);
	border-radius:5px;
}
QPushButton#btn_register:hover{
}
QPushButton#btn_register:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 255);
}
font: 700 36pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Sign up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_login">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>530</y>
      <width>371</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#btn_login{
	color: rgb(0, 99, 177);
	border-radius:5px;
}
QPushButton#btn_login:hover{
}
QPushButton#btn_login:pressed{
	padding-left:5px;
	padding-top:5px;
	background-color:rgba(105, 118, 132, 255);
}
font: 700 36pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Already have an account ?</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>280</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;antipasto&quot;;
background-color:rgba(0, 0, 0, 0); border:none;
border-bottom: 2px solid rgba(0, 99, 177, 255); color:rgba(156, 194, 213, 230);
padding-bottom:7px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_password">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>340</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;antipasto&quot;;
background-color:rgba(0, 0, 0, 0); border:none;
border-bottom: 2px solid rgba(0, 99, 177, 255); color:rgba(156, 194, 213, 230);
padding-bottom:7px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_conf_pwd">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>410</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;antipasto&quot;;
background-color:rgba(0, 0, 0, 0); border:none;
border-bottom: 2px solid rgba(0, 99, 177, 255); color:rgba(156, 194, 213, 230);
padding-bottom:7px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Confirm password</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
